modules/page/html/render/public.js

```diff
@@ -78,9 +80,14 @@ exports.create = function (api) {
           return isSubscribed || id === author || following().has(author)
         }
       },
+      observableFilter: computed([filters], function(filterFollowing) {
+        return {
+          filter: (msg) => !(filterFollowing && getType(msg) === 'contact')
```

Comment:
You can also add the `passthru` option to prevent flattening:

```js
computed([filters], ..., {
  passthru: true
})
```

Reply:
I overengineered this here, using your suggestion for public.js calling refresh I don't need this at all. Much better!